community/shell/src/main/java/org/neo4j/shell/impl/SystemOutput.java

```diff
@@ -35,12 +36,22 @@
 {
 	private PrintWriter out;
 
-    public SystemOutput() {
-        try {
-            out = new PrintWriter(new OutputStreamWriter(System.out,"UTF-8"));
-        } catch (UnsupportedEncodingException e) {
-            System.err.println("Unsupported encoding UTF-8, using "+Charset.defaultCharset()+", error: "+e.getMessage());
-            out = new PrintWriter(new OutputStreamWriter(System.out, Charset.defaultCharset()));
+    public SystemOutput()
+    {
+        this( System.out );
+    }
+
+    public SystemOutput( OutputStream out )
+    {
+        try
+        {
+            this.out = new PrintWriter( new OutputStreamWriter( out, "UTF-8" ) );
```

Comment:
StandardCharsets.UTF_8 should help to avoid `UnsupportedEncodingException` handling